mender-convert.sh

```diff
@@ -5,28 +5,36 @@ cat << EOF
 
 Mender conversion tool
 
-A tool for taking an existing embedded image (Debian, Ubuntu, Raspbian, etc) and
-convert it to a Mender image by restructuring partition table and adding
+A tool for taking an existing embedded image (Debian, Ubuntu, Raspbian, etc)
+and convert it to a Mender image by restructuring partition table and adding
 necessary files.
 
-Usage: $0 [prepare_image | make_sdimg | install_mender | install_bootloader | make_all] [options]
+Usage: $0 [raw-disk-image-shrink-rootfs |
```

Comment:
Since there are so many actions, would rather say

Usage: $0 ACTION [options]

Reply:
I prefer using COMMAND instead of ACTION. As you noticed it is more common for CLI. Even **mender-artifact** uses such notation.

Reply:
OK, let's use COMMAND! :+1: